examples/imageviewer/gradle.properties

```diff
@@ -1,15 +1,13 @@
 kotlin.code.style=official
 xcodeproj=iosApp
 android.useAndroidX=true
-org.gradle.jvmargs=-Xmx3g
+org.gradle.jvmargs=-Xmx8g
 
-# TODO: enable configuration cache once we update on kotlin 2.0
-# see https://youtrack.jetbrains.com/issue/KT-64851 (k/wasm issue)
-org.gradle.configuration-cache=false
+org.gradle.configuration-cache=true
 org.gradle.caching=true
 org.jetbrains.compose.experimental.jscanvas.enabled=true
 org.jetbrains.compose.experimental.macos.enabled=true
 org.jetbrains.compose.experimental.wasm.enabled=true
-kotlin.version=1.9.24
+kotlin.version=2.0.0-RC2
```

Comment:
Why not `2.0.0`?

Reply:
Typo. I tried to see if the issue with k/wasm present on RC. 
There is a TODO in the description. So the PR is not complete yet